selfdrive/car/tests/test_docs.py

```diff
@@ -27,10 +24,16 @@ def test_generator(self):
     assert generated_cars_md == current_cars_md, "Run selfdrive/car/docs.py to update the compatibility documentation"
 
   def test_docs_diff(self):
-    dump_path = os.path.join(BASEDIR, "selfdrive", "car", "tests", "cars_dump")
-    dump_car_docs(dump_path)
-    print_car_docs_diff(dump_path)
-    os.remove(dump_path)
+    new_path = CARS_MD_OUT
+
+    result = subprocess.run(['selfdrive/debug/print_docs_diff.py', '--path', new_path], capture_output=True, text=True)
```

Comment:
`FAILED test_docs.py::TestCarDocs::test_docs_diff - FileNotFoundError: [Errno 2] No such file or directory: 'selfdrive/debug/print_docs_diff.py'`

You should use imports

Reply:
Just reverted `test_docs.py`